_layouts/project-index.html

```diff
@@ -3,23 +3,57 @@
 ---
 
 <div class="project-index">
-
   <div class="project-map-area">
-
     <div id="map">
       <div id="loading-map">
         <center><p class="loader w40 h40 mb20 red-border"></p></center>
-        <span>Loading map...</span>
+        <span>Loading Map</span>
       </div>
-      <div id="hover-country" class="center-hover-map-area"></div>
-      <div class="our-work-map-legend">
-        <div class="legend-item projects">
-          <p>Projects</p>
+      <div id="hover-details" class="center-hover-map-area"></div>
+      <div id = 'country-legend-details' class="center-hover-map-area">
+        <div id = 'prefix-legend' class="legend-prefix">
+          <p>Countries with </p>
+        </div>
+        <div id = 'projects-legend'>
+          <p><img src= 'images/dot-red.png' width="9" height="9"/> <span style="color:#D73F3F"> Projects</span> </p>
+        </div> &nbsp;
+        <div id = 'active-legend'>
+            <p><img src= 'images/dot-active.png' width="10" height="10"/><span  style="color:#D73F3F"> Active tasks</span></p>
+        </div>&nbsp;
+        <div id = 'member-legend'>
+            <p><img src= 'images/dot-blue.png' width="7" height="7"/> <span style="color:#929DB3"> Members</span></p>
         </div>
-        <div class="legend-item members">
-          <p>Members</p>
+        <!-- <div id = 'project-area-legend' class='center-hover-map-area hide'> -->
+            <div id = 'cluster-legend' class = "hide">
```

Comment:
@ramyaragupathy Padding isn't completely even. If we use `<p>` tags instead we should get the same padding as the legend for the active countries.

Reply:
This was a good exercise for me in understanding padding details of different html elements 😸  Like you suggested, I've wrapped up all of the details into one `<p>` element